homeassistant/components/esphome/voice_assistant.py

```diff
@@ -92,6 +93,16 @@ def __init__(
         self._tts_done = asyncio.Event()
         self._tts_task: asyncio.Task | None = None
 
+    @property
+    def is_running(self) -> bool:
+        """True if the the UDP server is started and hasn't been stopped."""
+        return (
+            self.started
+            and (not self.stopped)
+            and (self.transport is not None)
+            and (not self.transport.is_closing())
```

Comment:
Are these needed? Wouldn't `self.stopped` set to true the moment the transport is closing

Reply:
I simplified things a bit in this direction and changed `stopped` to `stop_requested`